### PR TITLE
fix: it is possible to Occur OutOfCpu， when exist some pods including init container

### PR DESCRIPTION
**Pull request by huone1**
Signed-off-by: huone1 <huwanxing@huawei.com>
The Pod's effective request/limit:
image

issue: #1363

### CODE REVIEW

**Comment by huone1** [comment]
fix #1363

File: pkg/scheduler/api/job_info.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func getJobID(pod *v1.Pod) JobID {
 
 // NewTaskInfo creates new taskInfo object for a Pod
 func NewTaskInfo(pod *v1.Pod) *TaskInfo {
-	req := GetPodResourceWithoutInitContainers(pod)
+	req := GetPodResourceRequest(pod)
```
Comment (Thor-wl) [inline pkg/scheduler/api/job_info.go:107]: The relate issue is https://github.com/volcano-sh/volcano/issues/1333, which makes clear analysis about the the root reason. We should get the max(<init container resources>), sum(<containers in pod>).

Comment (Thor-wl) [reply pkg/scheduler/api/job_info.go:107]: Well, this fix is right.

Comment (k82cn) [reply pkg/scheduler/api/job_info.go:107]: /lgtm

**Pull request merged**